neurom/ezy/neuron.py

```diff
@@ -110,6 +112,12 @@ def __init__(self, neuron, iterable_type=np.array):
         super(Neuron, self).__init__(neuron.soma, neuron.neurites, neuron.name)
         self._iterable_type = iterable_type
 
+    def __eq__(self, other):
+        return False if not isinstance(self, type(other)) else \
+               all(self._compare_neurites(other, ttype) for ttype in
```

Comment:
That makes sense. I wonder if there are other ways to segment the neurons further. I'll merge this.